Content/posts/update-02-october-2020.md

```diff
@@ -0,0 +1,100 @@
+repository](https://github.com/swiftwasm/blog.swiftwasm.org).
+
+## SwiftWasm Team Updates
+
+We would like to welcome [@yonihemi](https://github.com/yonihemi) to the SwiftWasm team who joined
+us in the beginning of October. After the previous contributions he made to
+[`carton`][https://carton.dev] it made perfect sense to expand our team. As always, we invite
+everyone to contribute to any of our repositories, and it doesn't require much prior experience with
+SwiftWasm if any at all. Bug fixes, feature additions, improved documentation and related changes
+are very much appreciated and allow our ecosystem to grow even more!
+
+## JavaScriptKit
+
+[JavaScriptKit](https://github.com/swiftwasm/JavaScriptKit) had a few important updates in October
+so far. Most importantly, now that [#91](https://github.com/swiftwasm/JavaScriptKit/pull/91) by
+[@kateinoigakukun](https://github.com/kateinoigakukun) was merged, JavaScriptKit no longer uses
+unsafe flags in its `Package.swift`. The use of unsafe flags was a big problem for us, as it breaks
+dependency resolution due to strict checks that SwiftPM applies. If any package in your dependency
+tree contains unsafe flags, you can no longer depend on its semantic version, or a semantic version
+of any other package that depends on it.
+
+So far we were able to work around that with a hardcoded check for JavaScriptKit in our fork of
+SwiftPM. This was obviously a very ugly hack, but the biggest downside of that approach was that
+you couldn't depend on any package that had a semantic version dependency on JavaScriptKit in
+upstream Swift toolchains. That meant that libraries like [Tokamak](https://tokamak.dev) could not
+be built for macOS or Linux. And while the WebAssembly DOM renderer in Tokamak is the most useful
+module right now, this prevented us from working static HTML rendering in Tokamak on macOS and
+Linux.
+
+Another issue we had with JavaScriptKit is [the naming of `JSValueConstructible` and
+`JSValueConvertible`](https://github.com/swiftwasm/JavaScriptKit/issues/87) protocols. These
+protocols are used for conversions between `JSValue` references and arbitrary Swift values. In
+practice it wasn't always clear which of these protocols was responsible for a specific conversion.
+After some deliberation, these were renamed to `ConstructibleFromJSValue` and `ConvertibleToJSValue`
+respectively in [#88](https://github.com/swiftwasm/JavaScriptKit/pull/88).
+
+[An issue was raised](https://github.com/swiftwasm/JavaScriptKit/issues/97) by
+[@yonihemi](https://github.com/yonihemi) this week on our JavaScriptKit repository about `i64`
+Wasm function return type support in Safari. The reason for it is that [Safari is the only major
+browser](https://webassembly.org/roadmap/) that doesn't support
+[Wasm `i64` to `BigInt` conversion](https://github.com/WebAssembly/JS-BigInt-integration).
+Unfortunately, there is a plenty of APIs that do require this conversion to work, and it's unclear
```

Comment:
Possibly change to
```suggestion
Unfortunately, there is a plethora of APIs that require this conversion to work, and it's unclear
```
or
```suggestion
Unfortunately, there are (plenty | lots | many) of APIs that require this conversion to work, and it's unclear
```

Reply:
I agree with @carson-katri about “many:”

```suggestion
Unfortunately, there are many APIs that require this conversion to work, and it's unclear
```